app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+	validates :name, presence: true, length: { maximum: 50 }
+	validates :email, presence: true, length: { maximum: 235 }
```

Comment:
1. 2 space indentation
2. I found that, you have installed "config" gem. Can you use it to manage your constant.